dist/tools/renode/run-renode.sh

```diff
@@ -11,41 +11,90 @@
 # $image_file          Full path to the image file (see $IMAGE_FILE below)
 #
 # Global environment variables used:
-# RENODE:              Renode command name, default: "renode"
-# RENODE_CONFIG:       Renode configuration file name,
-#                      default: "${BOARDSDIR}/${BOARD}/dist/board.resc"
-# RENODE_BIN_CONFIG:   Renode intermediate configuration file name,
-#                      default: "${BINDIR}/board.resc"
+# RENODE:                       Renode command name, default: "renode"
+# RENODE_BOARD_CONFIG:          Renode configuration file name,
+#                               default: "${RIOTBOARD}/${BOARD}/dist/board.resc"
+# RENODE_BOARD_BIN_CONFIG:      Renode intermediate configuration file name,
+#                               default: "${BINDIR}/board.resc"
+# RENODE_BOARD_REPL:            Renode board repl file name,
+#                               default: "${RIOTBOARD}/${BOARD}/dist/board.repl"
+# RENODE_BOARD_BIN_REPL:        Renode intermediate board repl file name,
+#                               default: "${BINDIR}/board.repl"
+# RENODE_CPU_REPL:              Renode cpu repl file name,
+#                               default: "${RIOTCPU}/${CPU}/dist/cpu.repl"
+# RENODE_CPU_BIN_REPL:          Renode intermediate cpu repl file name,
+#                               default: "${BINDIR}/cpu.repl"
 #
 # @author       Bas Stottelaar <basstottelaar@gmail.com>
+# @author       Philipp-Alexander Blum <philipp-blum@jakiku.de>
+
+# Default path to Renode board configuration file
+: ${RENODE_BOARD_CONFIG:=${RIOTBOARD}/${BOARD}/dist/board.resc}
+# Default path to Renode board intermediate configuration file
+: ${RENODE_BOARD_BIN_CONFIG:=${BINDIR}/board.resc}
+# Default path to Renode board repl file
+: ${RENODE_BOARD_REPL:=${RIOTBOARD}/${BOARD}/dist/board.repl}
+# Default path to Renode board intermediate configuration file
+: ${RENODE_BOARD_BIN_REPL:=${BINDIR}/board.repl}
+
+# Default path to Renode cpu configuration file
+: ${RENODE_CPU_REPL:=${RIOTCPU}/${CPU}/dist/cpu.repl}
+# Default path to Renode cpu intermediate configuration file
+: ${RENODE_CPU_BIN_REPL:=${BINDIR}/cpu.repl}
 
-# Default path to Renode configuration file
-: ${RENODE_CONFIG:=${BOARDSDIR}/${BOARD}/dist/board.resc}
-# Default path to Renode intermediate configuration file
-: ${RENODE_BIN_CONFIG:=${BINDIR}/board.resc}
 # Default Renode command
-: ${RENODE:=renode}
+if [ ! -f "${RENODE}" ]; then
```

Comment:
Isn't `${RENODE:=renode}` doing the same?

Reply:
I guess so. I am not that good in shell scripting. Haven't seen that before.